net-p2p/retroshare/retroshare-0.6.9999.ebuild

```diff
@@ -14,131 +14,101 @@ LICENSE="GPL-2 GPL-3 Apache-2.0 LGPL-2.1"
 SLOT="0"
 KEYWORDS=""
 
-IUSE="cli feedreader gnome-keyring +gui voip"
+IUSE="cli control-socket gnome-keyring +gui settings-api +sqlcipher +webui"
 REQUIRED_USE="
 	|| ( cli gui )
-	feedreader? ( gui )
-	voip? ( gui )"
+	settings-api? ( || ( control-socket webui ) )"
 
 RDEPEND="
 	app-arch/bzip2
-	dev-db/sqlcipher
-	dev-libs/openssl:0
-	dev-qt/qtcore:5
-	dev-qt/qtmultimedia:5
-	dev-qt/qtnetwork:5
-	dev-qt/qtprintsupport:5
-	dev-qt/qtscript:5
-	dev-qt/qtxml:5
-	net-libs/libmicrohttpd
-	net-libs/libupnp:0
+	dev-libs/openssl:0=
+	>=dev-libs/rapidjson-1.1.0
+	net-libs/libupnp:0=
 	sys-libs/zlib
+	control-socket? ( dev-qt/qtnetwork:5 )
 	gnome-keyring? ( gnome-base/libgnome-keyring )
-	feedreader? (
-		dev-libs/libxml2
-		dev-libs/libxslt
-		net-misc/curl
-	)
 	gui? (
-		dev-qt/designer:5
+		dev-qt/qtcore:5
+		dev-qt/qtmultimedia:5
+		dev-qt/qtnetwork:5
+		dev-qt/qtprintsupport:5
+		dev-qt/qtscript:5
+		dev-qt/qtxml:5
 		dev-qt/qtgui:5
 		dev-qt/qtwidgets:5
 		dev-qt/qtx11extras:5
 		x11-libs/libX11
 		x11-libs/libXScrnSaver
 	)
-	voip? (
-		media-libs/opencv[-qt4(-)]
-		media-libs/speex
-		virtual/ffmpeg[encode]
-	)"
-DEPEND="${RDEPEND}
-	virtual/pkgconfig"
-
-src_prepare() {
-	local dir
-
-	sed -i \
-		-e "s|/usr/lib/retroshare/extensions6/|/usr/$(get_libdir)/${PN}/extensions6/|" \
-		libretroshare/src/rsserver/rsinit.cc \
-		|| die "sed on libretroshare/src/rsserver/rsinit.cc failed"
+	settings-api? ( dev-qt/qtcore:5 )
+	sqlcipher? ( dev-db/sqlcipher )
+	!sqlcipher? ( dev-db/sqlite )
```

Comment:
Sorry if my comment was unclear but please specify slot `:3` explicitly. We used to have SQLite2 in the same name.